forge-gui/res/cardsfolder/upcoming/phyrexian_vindicator.txt

```diff
@@ -0,0 +1,11 @@
+Name:Phyrexian Vindicator
+ManaCost:W W W W
+Types:Creature Phyrexian Horror
+PT:5/5
+K:Flying
+R:Event$ DamageDone | ActiveZones$ Battlefield | ValidTarget$ Card.Self | ReplaceWith$ DBImmediateTrigger | PreventionEffect$ True | AlwaysReplace$ True | Description$ If damage would be dealt to CARDNAME, prevent that damage.
+SVar:DBImmediateTrigger:DB$ ImmediateTrigger | Execute$ TrigDamage | RememberSVarAmount$ X | TriggerDescription$ When damage is prevented this way, CARDNAME deals that much damage to any other target.
+SVar:TrigDamage:DB$ DealDamage | ValidTgts$ Creature.Other,Player.Other,Planeswalker.Other | TgtPrompt$ Select any other target | NumDmg$ Y
```

Comment:
ValidTgts$ Creature.Other,Player,Planeswalker.Other (sorry for the bad advice :p )